XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -279,9 +284,12 @@ private void GetButtonSize(object sender, RoutedEventArgs e)
             AnimatedBtnMargin = AnimatedBtn.Margin;
         }
 
-        private void Animated_ScrollViewer_ViewChanging(object sender, ScrollViewerViewChangingEventArgs e)
+        private void Animated_ScrollViewer_ViewChanged(object sender, ScrollViewerViewChangedEventArgs e)
```

Comment:
Why the change (not against it, just curious)?

Reply:
ViewChanging does not fire once scrolling is done but rather in between. So if you scroll fast, the item in center and the displayed color are actually different (sometimes it isn't even on the screen anymore):

![Selected color not matching the visually selected color](https://user-images.githubusercontent.com/16122379/122806696-ec066180-d2ca-11eb-8b9e-67b974a4d03a.png)